ra/ra.go

```diff
@@ -780,7 +780,14 @@ func (ra *RegistrationAuthorityImpl) checkLimits(ctx context.Context, names []st
 
 // UpdateRegistration updates an existing Registration with new values.
 func (ra *RegistrationAuthorityImpl) UpdateRegistration(ctx context.Context, base core.Registration, update core.Registration) (reg core.Registration, err error) {
-	base.MergeUpdate(update)
+	changed := base.MergeUpdate(update)
+	if !changed {
+		// If merging the update didn't actually change the base then our work is
+		// done, we can return before calling ra.SA.UpdateRegistration since theres
+		// nothing for the SA to do
+		reg = base
```

Comment:
This already used naked  returns but for readability could you just switch it to a `return base, nil`?